src/types.ts

```diff
@@ -95,3 +96,79 @@ export enum Flavor {
   Elasticsearch = 'elasticsearch',
   OpenSearch = 'opensearch',
 }
+
+// TODO add raw data and logs
```

Comment:
I assume this is a not in this commit todo?

Reply:
made a ticket for this and put it in our backlog https://github.com/grafana/opensearch-datasource/issues/149